File: WebSolution/UIApp/src/app/leaves/Models/RoleBasedLeave.ts
export class RoleBasedLeave{
    
    constructor(
        public id:number,
        public roleId:number,
        public sick:number,
        public casual:number,
        public paid:number,
        public unpaid:number,
        public role:string,
        public financialYear:number
        ){}
}

File: Solution/WebSolution/Layered/TFLOBL/Entities/LeaveMgmt/RoleBasedLeave.cs
﻿namespace Transflower.TFLPortal.TFLOBL.Entities.LeaveMgmt;

public class RoleBasedLeave
{
    public int Id { get; set; }
    public int RoleId { get; set; }
    public int Sick { get; set; }
    public int Casual { get; set; }
    public int Paid { get; set; }
    public int Unpaid { get; set; }
    public int FinancialYear { get; set; }
}
